.vsts-ci-templates/build.yml

```diff
@@ -1,27 +1,21 @@
 parameters:
   cmake_args: ''
-  artifact_name: ''
 
 steps:
+- script: |
+    cat /proc/cpuinfo
+  displayName: /proc/cpuinfo
+  condition: succeededOrFailed()
+
 - task: CMake@1
   displayName: CMake
   inputs:
     cmakeArgs: '-GNinja -DTARGET=sgx .. ${{ parameters.cmake_args }}'
 
-- script: ninja
-  displayName: Ninja
-  workingDirectory: build
-
-- script: ./tests.sh -N
-  displayName: Create Python venv
+- script: cmake .. -L
```

Comment:
Can this be combined with the main cmake invocation?

Reply:
Oh yeah, of course it can!